state/applicationoffers_test.go

```diff
@@ -476,7 +491,7 @@ func (s *applicationOffersSuite) TestUpdateApplicationOfferNotFound(c *gc.C) {
 	owner := s.Factory.MakeUser(c, nil)
 	_, err := sd.UpdateOffer(crossmodel.AddApplicationOfferArgs{
 		OfferName:       "hosted-mysql",
-		ApplicationName: "foo",
+		ApplicationName: "mysql",
```

Comment:
This test began to fail for a different reason (application "foo" doesn't exist), due to the added validation checks. We need to ensure the application actually exists but the offer does not.